sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1479,6 +1479,42 @@ class Analyzer(
       // Skip the having clause here, this will be handled in ResolveAggregateFunctions.
       case h: UnresolvedHaving => h
 
+      case a: Aggregate =>
```

Comment:
there is a `case a @ Aggregate(groupingExprs, aggExprs, appendColumns: AppendColumns)` above, please merge that  into this case.

Reply:
> there is a `case a @ Aggregate(groupingExprs, aggExprs, appendColumns: AppendColumns)` above, please merge that into this case.

Updated